.stylelintrc.json

```diff
@@ -0,0 +1,22 @@
+{
+  "extends": ["stylelint-config-standard"],
+  "plugins": ["stylelint-scss", "stylelint-csstree-validator"],
+  "rules": {
+    "at-rule-no-unknown": null,
+    "scss/at-rule-no-unknown": [
+      true,
+      {
+        "ignoreAtRules": [
+          "tailwind",
+          "apply",
+          "variants",
+          "responsive",
+          "screen"
+        ]
+      }
+    ]
+  },
+  "csstree/validator": true,
+  "ignoreFiles": ["build/**", "dist/**", "**/reset*.css", "**/bootstrap*.css"]
+}
+
```

Comment:
## 1. 
![pr-title](https://user-images.githubusercontent.com/93039359/184694119-bff8d371-b374-4a4f-b115-1c6b8799584a.png)

- [ ] Your PR has a title that is not so descriptive enough. Because **Feature branch** is not describing what are you doing in the project. PR title fully describes what is happening in this branch. You could use **Setup Hello Microverse Project** for your PR title. :+1: